TestsPython/tests/main_menu_page_test.py

```diff
@@ -3,7 +3,7 @@
 
 
 class TestMainMenuPage(TestBase):
-    def setUp(self):
+    def setup(self):
```

Comment:
If the newest version of pytest is used, it is possible to have warnings about deprecation of `setup`. 

Please see https://docs.pytest.org/en/stable/deprecations.html#setup-teardown  Deprecated since version 7.2.

Reply:
I would just use `conftest` for fixtures instead of this `setup` - see the comment from @robert-altom for that.